doc/api/net.md

```diff
@@ -902,6 +919,13 @@ For TCP connections, available `options` are:
   **Default:** `false`.
 * `keepAliveInitialDelay` {number} If set to a positive number, it sets the initial delay before
   the first keepalive probe is sent on an idle socket.**Default:** `0`.
+* `autoDetectFamily` {boolean}: Enables the Happy Eyeballs connection algorithm.
```

Comment:
Assuming all addresses are functioning, it's not so much about _detecting_ address families, but more about selecting the fastest one. How about `autoSelectFamily`, `autoFastestAddress`, or something in that direction? (I am not good at coming up with names right now.)